### PR TITLE
CART-148 added hapi-auth-header

### PR DESCRIPTION
**Pull request by RomanFedytskyi**

### CODE REVIEW

File: dev/server.js [hunk 1]
```diff
@@ -120,21 +119,21 @@ function registerACL(server) {
...
-        server.auth.strategy('simple', 'bearer-access-token', {
+         server.auth.strategy('simple', 'auth-header', {
           accessTokenName: 'X-CART-Token',    // optional, 'access_token' by default
-          validateFunc: function (token, callback) {
-
+          validateFunc: function (tokens, callback) {
             // For convenience, the request object can be accessed
             // from `this` within validateFunc.
             var request = this;
+            var headerName = tokens['X-CART-Token'];
```
Comment (dmitrij-borchuk) [inline dev/server.js]: 'headerName' consist token, something wrong

File: package.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "gulp": "^3.9.1",
     "hapi": "^13.5.0",
     "hapi-auth-bearer-token": "^4.2.1",
```
Comment (dmitrij-borchuk) [inline package.json]: May be "hapi-auth-bearer-token" should be deleted?

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (RomanFedytskyi) [reply package.json]: fixed

File: dev/server.js (hunk 1 above)
Comment (RomanFedytskyi) [reply dev/server.js]: fixed

**Review by dmitrij-borchuk — APPROVED** [review]

**Pull request merged**